fixed output connections (#11)
ShvaykaD/thingsboard.github.io: _includes/docs/user-guide/rule-engine-2-0/ce-enrichment-nodes.md

![image](/images/user-guide/rule-engine-2-0/nodes/enrichment-calculate-delta-config.png)

**Output connections**
* **Success:**
  * If key configured via **Input value key** parameter is present in the incoming message.
* **Other:**
  * If key configured via **Input value key** parameter is not present in the incoming message.
* **Failure:**

>   * If the **Tell Failure if delta is negative** is set and the delta calculation returns negative value.

Reviewer: ```suggestion
  * If **Tell Failure if delta is negative** is enabled and delta calculation returns a negative value.
```

Aggregated data will be placed in outgoing message's metadata as a simple key-value entry, like in the **First** or **Last** strategies.

**Output connections**
* **Success:**
  * Time series data is found, and is placed in the outgoing message's metadata. 
  * If no time series data is found, nothing is placed in the outgoing message's metadata.
* **Failure:**

>   * **Use dynamic interval** is toggled, and one or both templates are not found in the incoming message.

Reviewer: ```suggestion
  * **Use dynamic interval** is enabled, and one or both templates are not found in the incoming message.
```
